### PR TITLE
Interactions (additional description)

### PR DESCRIPTION
**Pull request by karolamik13**
I added information about the hpb code in README and in docx.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -72,3 +72,19 @@ with ProDy. The original CE method was developed by Ilya Shindyalov and Philip
 Bourne. The Python version which is used by ProDy is developed by Jason Vertrees 
 and available under the New BSD license. 
 
+Hbp module: The calculation of hydrophobic interactions, solvent accessible surface 
+area (SASA) and volume for each residue is using geometric methods based on the 
+information of the atoms in the molecule. The methods have been programmed in C++ 
+and can be compiled as a python module “hpb.so” which is then used by ProDy.
+Files for compilation are localized at ~/prody/proteins/hpbmodule folder and
+required C++ and Fortran compiler. After compilation hpb.so file can be
+storage in ~/prody/proteins folder in ProDy or in the local directory which
```
Comment (jamesmkrieger) [inline README.rst:81]: stored

File: README.rst [hunk 2]
```diff
@@ -72,3 +72,19 @@ with ProDy. The original CE method was developed by Ilya Shindyalov and Philip
...
+is used to perform calulations. The precompiled versions for Python 2.7,
+3.8, 3.9, and 3.10 are availabe in ~/prody/proteins/hpbmodule. The user can
```
Comment (jamesmkrieger) [inline README.rst:83]: just prody/proteins without ~

File: README.rst [hunk 3]
```diff
@@ -72,3 +72,19 @@ with ProDy. The original CE method was developed by Ilya Shindyalov and Philip
...
+choose the correct version of hpb.so and copy to the ~/prody/proteins or
+local directory.
+C++ code of hpb.so was developed by Xin Cao and Fortran code by Xin Cao, 
+Michelle H. Hummel, Bihua Yu, and Evangelos A. Coutsias (License in 
+~/prody/proteins/hpbmodule folder). Details of the method can be found 
+in the Supplementary Material of InSty manuscript 
+(soon will be submited for publication).
```
Comment (jamesmkrieger) [inline README.rst:90]: submitted with 2 t

**Review by jamesmkrieger — CHANGES_REQUESTED** [review]
Looks good. Just a few minor things

**Comment by karolamik13** [comment]
All changes added. Thank you, James.

**Pull request merged**